lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2559,3 +2566,17 @@ definitions:
         pattern: '^(\d{2,3})$'
         example: '01'
         x-nullable: false
+
+  imei:
+    description: IMEI value
+    type: object
+    required:
+      - tac
+    properties:
+      tac:
+        $ref: '#/definitions/tac'
```

Comment:
note that this is not the same as tracking area code, this is a type allocation code.

Reply:
Thanks for the catch. Fixed that.